src/FileSigChecker.cs

```diff
@@ -166,17 +150,17 @@ public class HexDump
         /// <returns>string</returns>
         public static string GetHex(string fileName)
         {
-            string outHex = "";
+            var outHex = "";
             try
             {
                 if (File.Exists(fileName))
                 {
                     using (var stream = File.OpenRead(fileName))
                     {
-                        int size = stream.Length < 50 ? (int)stream.Length : 50;
-                        byte[] buffer = new byte[size];
+                        var size = stream.Length < 50 ? (int) stream.Length : 50;
+                        var buffer = new byte[size];
                         var read = stream.Read(buffer, 0, size);
-                        outHex = Hex(buffer, 50);
+                        outHex = Hex(buffer, size);
```

Comment:
I changed the hard coded 50 here to be size as seems appropriate for bytes per line